tutorial01_load_vulkan/app/build.gradle

```diff
@@ -15,26 +15,26 @@
 apply plugin: 'com.android.application'
 
 android {
-    compileSdkVersion 31
-    ndkVersion '22.1.7171670'
+    compileSdkVersion 34
+    ndkVersion '25.2.9519653'
 
     defaultConfig {
         applicationId "com.android.example.vulkan.tutorials.one"
-        minSdkVersion 24
-        targetSdkVersion 31
+        minSdkVersion 26
+        targetSdkVersion 34
```

Comment:
In the readme it looks like we're targeting 31 for x86-64, and 34 for everything else. Should this be 31? Or do we need a per-ABI version here?